ctlearn/default_models/single_cnn.py

```diff
@@ -8,7 +8,10 @@ def single_cnn_model(data, model_params):
 
     # Load neural network model
     network_input_img = tf.keras.Input(shape=data.img_shape, name=f"images")
-    network_input_param = tf.keras.Input(shape=data.prm_shape, name=f"parameters")
+    flag_prm = 0
```

Comment:
There is no need for this integer. You could use consistent `data.prm_shape != None` instead.

Reply:
I thought it would be quicker to check the flag_prm than to check data.prm_shape again in the if statement on line #28

Reply:
Done